plugins/source/okta/codegen/recipes/applications.go

```diff
@@ -0,0 +1,64 @@
+package recipes
+
+import (
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/okta/okta-sdk-golang/v2/okta"
```

Comment:
https://pkg.go.dev/github.com/okta/okta-sdk-golang/v3/okta

Reply:
It doesn't look like v3 is released yet according to https://github.com/okta/okta-sdk-golang

Reply:
https://github.com/okta/okta-sdk-golang/releases/tag/v3.0.1

Reply:
They didn't update the readme 🤷

Reply:
Fixed in [`91c66d2` (#5118)](https://github.com/cloudquery/cloudquery/pull/5118/commits/91c66d25ce8b8cfc281981ac1d79fe203c90ec33)